### PR TITLE
New Package: Peltoche/lsd/0.23.1

### PR DESCRIPTION
**Pull request by neomax7**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94721)

### CODE REVIEW

**Comment by neomax7** [comment]
manifest v1.4 with zip

**Comment by neomax7** [comment]
v1.4 with zip

**Review by sitiom — CHANGES_REQUESTED** [review]
Add 32-bit executable: https://github.com/Peltoche/lsd/releases/download/0.23.1/lsd-0.23.1-i686-pc-windows-msvc.zip

**Comment by neomax7** [comment]
> Add 32-bit executable: https://github.com/Peltoche/lsd/releases/download/0.23.1/lsd-0.23.1-i686-pc-windows-msvc.zip

done~

File: manifests/p/Peltoche/lsd/0.23.1/Peltoche.lsd.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+InstallerLocale: en-US
+InstallerType: zip
+Installers:
+  - Architecture: x64
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: lsd-0.23.1-x86_64-pc-windows-msvc\lsd.exe
+        PortableCommandAlias: lsd
+    InstallerUrl: https://github.com/Peltoche/lsd/releases/download/0.23.1/lsd-0.23.1-x86_64-pc-windows-msvc.zip
+    InstallerSha256: 4666763AF99D7867AA9795101560BAEB053CCBA12BA99A906E9A197B70E756A5
```
Comment (sitiom) [inline manifests/p/Peltoche/lsd/0.23.1/Peltoche.lsd.installer.yaml:15]:
```suggestion
    InstallerSha256: 4666763AF99D7867AA9795101560BAEB053CCBA12BA99A906E9A197B70E756A5
    Dependencies:
      PackageDependencies:
        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

File: manifests/p/Peltoche/lsd/0.23.1/Peltoche.lsd.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - Architecture: x86
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: lsd-0.23.1-i686-pc-windows-msvc\lsd.exe
+        PortableCommandAlias: lsd
+    InstallerUrl: https://github.com/Peltoche/lsd/releases/download/0.23.1/lsd-0.23.1-i686-pc-windows-msvc.zip
+    InstallerSha256: E6AFF15F93A3AC7D183C1F4BBAA8081FCBE1F4DC8E4E12248859C85449258566
```
Comment (sitiom) [inline manifests/p/Peltoche/lsd/0.23.1/Peltoche.lsd.installer.yaml:22]:
```suggestion
    InstallerSha256: E6AFF15F93A3AC7D183C1F4BBAA8081FCBE1F4DC8E4E12248859C85449258566
    Dependencies:
      PackageDependencies:
        - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Review by sitiom — CHANGES_REQUESTED** [review]
Requires VCRedist dependency

**Comment by sitiom** [comment]
Continued in #95485

**Pull request closed without merging**